src/components/TopBar/TopBar.vue

```diff
@@ -416,40 +408,13 @@ export default {
 		margin-right: var(--default-clickable-area);
 	}
 
-	&.in-call {
+	&[data-theme-dark="true"] {
 		right: 0;
 		border: none;
 		position: absolute;
 		top: 0;
-		left:0;
+		left: 0;
 		background-color: transparent;
-		display: flex;
-		flex-wrap: wrap;
```

Comment:
`flex-wrap` is needed for small screens (but you could move it to `.top-bar` class:
![image](https://github.com/nextcloud/spreed/assets/93392545/54fc66d8-2762-462b-90a1-91e9a68be06c)